src/main/java/mate/academy/bookstore/model/Category.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Getter
+@Setter
+@Entity
+@SQLDelete(sql = "UPDATE categories SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction("is_deleted = FALSE")
+@NoArgsConstructor
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name",
+            nullable = false)
+    private String name;
+    @Column(name = "description")
+    private String description;
+    @Column(name = "is_deleted",
+            nullable = false)
```

Comment:
not critical but in this case explicitly specifying column name is redundant as Hibernate will automatically map it (for example field name `isDeleted` will be mapped to `is_deleted` implicitly as default)
```suggestion
    @Column(nullable = false)
    private String name;
    private String description;
    @Column(nullable = false)
```